### DISCUSSION TITLE
deteals of of the dataset

### DISCUSSION THREAD

**Discussion by jopusun** [nanoporetech/bonito#283] [General]
I'm trying building my own model with data from:
bonito download --training

What i got is a folder named "dna_r9.4.1" with some .npy files
Can I get some deteals of the dataset?
Like species, from how many reads etc.

Thanks a lot!

**Comment by vellamike** [comment]
Hi @jopusun , this dataset is really there to aid in model development - it is chosen arbitrarily and its composition isn't important.It shouldn't be used for any purpose beyond testing software.

**Reply by jopusun** [reply]
Thanks for reply
The other thing I want to check did the dataset been shuffled?
Want to make sure if I take a part of data to train my model without shuffling is okay(The whole dataset is to big so I use a small dataset to make sure my model is working)
I'm afraid to get all my training data from some specific species and have bad impact on my result.

The last thing is that is it allow to published my result with those dataset, and how to site it properly?

Sincerely,
Jopu, Sun

**Reply by vellamike** [reply]
Bonito will shuffle after the first epoch so it doesn't really matter. I suggest you shuffle anyway though, there is no downside.

Yes you can use this data for your research, please cite it using the bonito project URL.
